### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by darakian** [tiny-conf/tiny-conf#3]
Hi, do you intend to make a new release with https://github.com/tiny-conf/tiny-conf/commit/1f7be78bc68927996647cd45b4367f8975a3ea05 ? If so is there a schedule for that release?

**Comment by maxleiko** [comment]
@darakian not really, if I were to do something it would be removing the package from the npm registry.

Why the question though ?  
I can see that noone is depending on this package (besides kevoree, which was a project I worked on, and is discontinued)

**Comment by darakian** [comment]
> Why the question though ?

I'm adding an advisory for your package to the github advisory DB and in writing it up I saw that there was a fix for the issue which had not been published. I wanted to see if there was intent to publish the fix or not so I could give better guidance to those that will get the advisory.

Ref: https://nvd.nist.gov/vuln/detail/CVE-2020-7724
